native/src/widget/scrollable.rs

```diff
@@ -700,34 +891,8 @@ pub fn draw<Renderer>(
                 ..bounds
             },
             |renderer| {
-                if is_scrollbar_visible {
-                    renderer.fill_quad(
-                        renderer::Quad {
-                            bounds: scrollbar.bounds,
-                            border_radius: style.border_radius,
-                            border_width: style.border_width,
-                            border_color: style.border_color,
-                        },
-                        style
-                            .background
-                            .unwrap_or(Background::Color(Color::TRANSPARENT)),
-                    );
-                }
-
-                if is_mouse_over
-                    || state.is_scroller_grabbed()
-                    || is_scrollbar_visible
-                {
-                    renderer.fill_quad(
-                        renderer::Quad {
-                            bounds: scrollbar.scroller.bounds,
-                            border_radius: style.scroller.border_radius,
-                            border_width: style.scroller.border_width,
-                            border_color: style.scroller.border_color,
-                        },
-                        style.scroller.color,
-                    );
-                }
+                draw_scrollbar(renderer, state.scrollbar_y.as_ref());
+                draw_scrollbar(renderer, state.scrollbar_x.as_ref());
```

Comment:
I don't think we need this function helper, since we are calling it only twice with completely different directions. We can just inline each one of the branches here.